app/store/utils.js

```diff
@@ -0,0 +1,12 @@
+import Immutable from 'immutable';
+
+export const callWithStore = (callback) => {
```

Comment:
this is good idea, but you combine here part of redux setup and reading from localStorage

better to left here only logic related to localStorage

like create class with methids

getInitialState
setInitialSTate

which will just return data from localStorage


and better to have all redux setup in one place